src/indicators/flooding/glofas/get_stations.py

```diff
@@ -0,0 +1,38 @@
+"""
+Script to turn csv of GloFAS stations into config format
+for aa toolbox. Input is country ISO2 code.
+"""
+import os
+import sys
+from pathlib import Path
+
+import pandas as pd
+
+DATA_DIR = Path(os.environ["AA_DATA_DIR"])
+FILENAME = (
+    DATA_DIR
+    / "private/exploration/glb/glofas/Qgis_World_outlet_202104_20210421.csv"
+)
+
+country_code = sys.argv[1]
```

Comment:
Would be more user friendly if you can also input lower case. But get that this is a quick script and not toolbox script